diff --git a/CMakeLists.txt b/CMakeLists.txt
index df45fb9d8..7188081c1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -314,14 +318,14 @@ if(WIN32)
 	# Include all dynamically linked runtime libraries such as MSVCRxxx.dll
 	include(InstallRequiredSystemLibraries)
 
-	if(RUN_IN_PLACE)
+	if(TRUE)
 		if(CMAKE_SIZEOF_VOID_P EQUAL 8)
 			set(CPACK_PACKAGE_FILE_NAME "${PROJECT_NAME}-${VERSION_STRING}-win64")
 		else()
 			set(CPACK_PACKAGE_FILE_NAME "${PROJECT_NAME}-${VERSION_STRING}-win32")
 		endif()
 
-		set(CPACK_GENERATOR ZIP)
+		set(CPACK_GENERATOR 7Z)
 
 	else()
 		set(CPACK_GENERATOR WIX)
diff --git a/util/buildbot/common.sh b/util/buildbot/common.sh
index 32434abdd..25b5b40f0 100644
--- a/util/buildbot/common.sh
+++ b/util/buildbot/common.sh
@@ -133,5 +133,8 @@ add_cmake_libs () {
 		-DLEVELDB_INCLUDE_DIR=$libdir/libleveldb/include
 		-DLEVELDB_LIBRARY=$libdir/libleveldb/lib/libleveldb.dll.a
 		-DLEVELDB_DLL=$libdir/libleveldb/bin/libleveldb.dll
+
+		-DBUILD_UNITTESTS=0
+		-DRUN_IN_PLACE=0
 	)
 }
