diff --git a/CMakeLists.txt b/CMakeLists.txt
index df45fb9..66bbc8d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -141,7 +141,7 @@ endif()
 
 if(WIN32)
 	set(SHAREDIR ".")
-	set(BINDIR "bin")
+	set(BINDIR ".")
 	set(DOCDIR "doc")
 	set(EXAMPLE_CONF_DIR ".")
 	set(LOCALEDIR "locale")
diff --git a/src/porting.cpp b/src/porting.cpp
index 81e9afa01..f3d417e5b 100644
--- a/src/porting.cpp
+++ b/src/porting.cpp
@@ -449,8 +449,8 @@ bool setSystemPaths()
 
 	std::string exepath(buf);
 
-	// Use ".\bin\.."
-	path_share = exepath + "\\..";
+	// Use ".\"
+	path_share = exepath;
 	if (detectMSVCBuildDir(exepath)) {
 		// The msvc build dir schould normaly not be present if properly installed,
 		// but its useful for debugging.
